src/core.py

```diff
@@ -117,10 +117,22 @@ def _get_module_folders(self, temp_repo):
         """
         paths = (
             os.path.join(temp_repo, path) for path in os.listdir(temp_repo)
-            if self._is_module_included(path)
+            if self._is_odoo_module(temp_repo, path) and self._is_module_included(path)
         )
         return (path for path in paths if os.path.isdir(path))
 
+    def _is_odoo_module(self, repo_path, file_name):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/numigi/gitoo/pullRequest?prid=4535071)